transactions/build.gradle

```diff
@@ -14,11 +14,22 @@
  * limitations under the License.
  */
 
+group = 'org.ehcache'
+
+apply plugin: EhOsgi
 apply plugin: EhDeploy
+apply plugin: EhPomMangle
+apply plugin: EhDocs
```

Comment:
This causes a build error on Java 1.6 - no idea why. I also believe it is not needed, the default `javadoc` task from the parent `build.gradle` should do what's needed. Having API and SPI javadoc jars at this level is just too much.